specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimapis.json

```diff
@@ -3395,7 +3399,8 @@
             "enum": [
               "swagger-link",
               "wadl-link",
-              "wsdl-link"
+              "wsdl-link",
+              "openapi-link"
```

Comment:
It would be great to have an example of how customer would provide YAML when creating an API.
https://github.com/Azure/azure-rest-api-specs/blob/71c04b82c5fd21293f99a0ebd01ddfefc8224f94/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimapis.json#L221